app/services/s3_handler.rb

```diff
@@ -44,6 +46,28 @@ def presigned_url_for_output(expires_in: 3600)
     raise Error.new(e)
   end
 
+  def presigned_url_for_input(key, content_type, job_id)
+    signer = Aws::S3::Presigner.new(client: @s3_client)
+
+    url = signer.presigned_url(
+      :put_object,
+      bucket: ENV.fetch('S3_BUCKET_NAME'),
```

Comment:
Is there a way to get the bucket name from what's been initialized?  Like maybe `@bucket.name`?